glossary.md

```diff
@@ -147,10 +150,13 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 ## I
 | English                    | Tiếng Việt         | Thảo luận tại |
 |----------------------------|--------------------|---------------|
+| implement | xây dựng / lập trình | |
```

Comment:
```suggestion
| implement | triển khai | |
```

Reply:
Bác @tanduong ở PR trước có nói, mình thấy đúng, là `triển khai` sẽ bị nhầm sang `deploy`, và bản thân từ `triển khai` cũng không sát nghĩa từ `implement` trong bối cảnh viết mã nữa.

Reply:
nãy nhầm thread nên mình bê comment qua đây
> @sudohainguyen: Mình thường dịch deploy là hiện thực hóa (tức từ code ra production), implement là triển khai (từ idea/pseudo sang code). Còn từ "xây dựng" dễ bị hiểu nhầm là build ấy.

Reply:
> Mình thường dịch deploy là hiện thực hóa (tức từ code ra production), implement là triển khai (từ idea/pseudo sang code). Còn từ "xây dựng" dễ bị hiểu nhầm là build ấy.

@sudohainguyen bạn comment nhầm bên mình bưng lại qua đây thảo luận nhé :)).
Haha tiếng việt của mình với bạn hiểu cũng ngược nhau rồi. Mình hiểu 'hiện thực hoá' là từ ý tưởng -> code, triển khai là từ demo -> production. 
'Xây dựng' ở đây hiểu bị hiểu nhầm sang build cũng không sao vì trong các ngữ cảnh `implement function abc` hiểu thành `build/write function abc` cũng đúng.

Reply:
hoặc mình nghĩ từ này nên flexible theo context của câu, có thể sẽ liệt kê các nghĩa cho người dịch chọn :nerd_face:

Reply:
Mình trước giờ cũng nghĩ vậy nên trước giờ chưa đề xuất thêm từ này vào glossary. Tình hình bây giờ có vẻ nhiều bạn dịch chữ này nhiều kiểu hiểu khác nhau nên @minhduc0711  mới đề xuất.

Reply:
Mình khá chắc mọi người đều hiểu `implement` theo 1 nghĩa thôi, là việc code thuật toán / phần mềm. Vấn đề chỉ là mọi người đang dùng các từ khác nhau cho cùng 1 thứ. 
`Xây dựng` nghe mang tính vĩ mô hơn (e.g. xây dựng phần mềm từ khâu thiết kế đến deploy), nên mình sẽ stick với `lập trình`.

Reply:
Trong cuốn "Nhập môn Lập trình" của trường KHTN thì từ này được dịch là `cài đặt`. Theo mọi người thì sao?